docs/source/reference/links/faster_rcnn.rst

```diff
@@ -4,64 +4,73 @@ Faster R-CNN
 .. module:: chainercv.links.model.faster_rcnn
 
 
+Detection Link
+--------------
+
+FasterRCNNVGG16
+~~~~~~~~~~~~~~~
+.. autoclass:: FasterRCNNVGG16
+
+
+Utility
```

Comment:
In `ssd.rst`, `Utility classes`. Please rename that (`Utility classes` -> `Utility`)